iamy/aws.go

```diff
@@ -214,22 +223,34 @@ func (a *AwsFetcher) populateIamData(resp *iam.GetAccountAuthorizationDetailsOut
 		}
 
 		for _, version := range policyResp.PolicyVersionList {
-			if *version.IsDefaultVersion {
-				doc, err := NewPolicyDocumentFromEncodedJson(*version.Document)
+			if !*version.IsDefaultVersion {
+				continue
+			}
+
+			doc, err := NewPolicyDocumentFromEncodedJson(*version.Document)
+			if err != nil {
+				return err
+			}
+
+			description := ""
+			if a.Mode == ModePull {
+				log.Printf("getPolicyDescription(%s)", *policyResp.Arn)
+				description, err = a.iam.getPolicyDescription(*policyResp.Arn)
```

Comment:
I am I right in thinking the description is only applied to the Policy, not the Policy Version. That means this only needs to be done once per policy, rather than once per Policy Version